tests/DependencyInjection/SchebTwoFactorExtensionTest.php

```diff
@@ -107,6 +107,30 @@ public function load_fullConfig_setConfigValues(): void
         $this->assertHasParameter(['127.0.0.1'], 'scheb_two_factor.ip_whitelist');
     }
 
+    /**
+     * @test
+     */
+    public function load_envVarBasedFullConfig_setConfigValues(): void
```

Comment:
This should be two test methods, one for the `true` case, one for the `false` case.